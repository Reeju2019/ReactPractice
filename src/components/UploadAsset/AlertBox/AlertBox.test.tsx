import { shallow } from "enzyme"
import ReactDOM from "react-dom"
import { AlertBox } from "."

const localeConfig = {
  uploadAsset_upload_msg_failure: "Hello world",
}

describe("UploadAsset Testing", () => {
  it("render without crashing", () => {
    const div = document.createElement("div")
    ReactDOM.render(
      <AlertBox alertMsg={localeConfig.uploadAsset_upload_msg_failure} />,
      div
    )
    ReactDOM.unmountComponentAtNode(div)
  })
  it("should render initial layout", () => {
    const component = shallow(
      <AlertBox alertMsg={localeConfig.uploadAsset_upload_msg_failure} />
    )
    expect(component.getElements()).toMatchSnapshot()
  })
})
