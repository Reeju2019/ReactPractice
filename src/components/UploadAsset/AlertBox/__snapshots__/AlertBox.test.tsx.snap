// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UploadAsset Testing should render initial layout 1`] = `
Array [
  <React.Fragment>
    <Col
      className="alert-box--active"
      md={12}
      xs={12}
    >
      Hello world
    </Col>
  </React.Fragment>,
]
`;
